test/unit/quicknxs/interfaces/data_handling/test_processing_workflow.py

```diff
@@ -0,0 +1,57 @@
+import os
```

Comment:
This tests looks more like an integration test than a unit test, don't you think?